src/kernel-install/60-ukify.install.in

```diff
@@ -132,9 +132,10 @@ def input_file_location(
     if root := os.getenv('KERNEL_INSTALL_CONF_ROOT'):
         search_directories = (root,)
     elif not search_directories:
-        # This is the default search path.
-        search_directories = ('/etc/kernel',
-                              '/usr/lib/kernel')
+        search_directories = ('/run/kernel',
+                              '/etc/kernel',
+                              '/usr/lib/kernel',
+                              '/usr/local/lib/kernel')
```

Comment:
the commit message should mention that usr/local is also added, not just run

Reply:
it's ok to support /usr/local/ but i am not sure we should document it much, because it's such a flawed concept.

Reply:
The commit message has:
> /usr/local/lib/kernel/ is used too, but not documented, as is done in other
places.

I didn't add `/usr/local/lib/` to documentation, because, as Lennart said, we generally don't document those paths.